### ISSUE TITLE
How to use Sliding Menu with an Activity that isn't FragmentActivity?

### ISSUE THREAD

**Issue by 12520054** [jfeinstein10/SlidingMenu#725]
Hi there,
I don't know how to use sliding menu with my activity. It isn't Fragmentactivity

**Comment by milos85vasic** [comment]
Obviously, it is working with ordinary activities (Activity class) and fragment activities as well (FragmentActivity). 

Cheers!
ps. Can you be more specific about the problem since this seems to work.
